lib/features/family/features/profiles/cubit/profiles_cubit.dart

```diff
@@ -41,6 +46,18 @@ class ProfilesCubit extends Cubit<ProfilesState> {
         fetchActiveProfile();
       },
     );
+    _cachedMembersSubscription =
+        _cachedMembersRepository.onCachedMembersChanged().listen(
+      (members) {
+        emit(
```

Comment:
I would create a method for this. You can reuse that down below as well.

Reply:
do you mean something like:
```
(members) {
        _emitCachedMembers(members);
      },
```
  ?